backend/index.js

```diff
@@ -1,13 +1,22 @@
-require('dotenv').load()
-global.Promise = require('bluebird')
+global.Promise = require('bluebird') // TODO: get rid of this since we're using babel?
```

Comment:
Add @SGrondin's comments to the comments:

- That [bluebird adds many convenient APIs](http://bluebirdjs.com/docs/api-reference.html) and that doing this makes it possible to fully take advantage of Sequelize's use of Bluebird
- Bluebird's promises [are better designed and have fewer issues than native ones](jamesknelson.com/are-es6-promises-swallowing-your-errors/)